packages/hub/routes/job-tickets.ts

```diff
@@ -3,13 +3,16 @@ import autoBind from 'auto-bind';
 import { query } from '@cardstack/hub/queries';
 import { ensureLoggedIn } from './utils/auth';
 import { inject } from '@cardstack/di';
+import shortUUID from 'short-uuid';
+import { JOB_TICKETS_STATE_DEFAULT } from '../db/migrations/20220610203119883_create-job-tickets';
```

Comment:
Alternatively, we could also remove the SQL default value, put this constant in the db query file, and have the db query file be responsible for adding it as a default where not specified.

Reply:
I originally had  `state: null` when creating the new job, allowing the database to set the default, since the Javascript doesn’t really need to know the string, but it caused Typescript problems. In 4bbf50 I have adjusted the model type to allow that, thanks.